opslevel.yml

```diff
@@ -0,0 +1,9 @@
+service:
+  name: que
+  owner: back_office_experience_box
+  description: High-performance job queue.
+  language: Ruby
+  tier: tier_1
+  lifecycle: pre-release
+  tags:
+  - key: type
```

Comment:
the value of this key is missing

Reply:
seems like you forgot to add the type value at the very end of the file
`value: <the type of this project>`
for example
  ```
- key: type
    value: service
```